peachjam/adapters/gazettes.py

```diff
@@ -98,6 +114,12 @@ def update_document(self, expression_frbr_uri):
                     defaults={"file": f"{file_path}", "source_url": source_url},
                 )
 
+            if hasattr(ga_gazette, "document_content"):

```

Comment:
I think this should come after the source file update, because now the source file block is no longer a part of the above if statement.

Updating the source file and the document content model are separate concerns.

Reply:
Thanks. Changed